coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetCommonController.java

```diff
@@ -68,7 +75,14 @@ public ModelAndView start(@RequestParam("budgetId") Long budgetId, @RequestParam
         if (auditActivated != null){
             form.setAuditActivated(Boolean.parseBoolean(auditActivated));
         }
-        return getModelAndViewService().getModelAndViewWithInit(form, ProposalBudgetConstants.KradConstants.BUDGET_DEFAULT_VIEW);
+        ModelAndView result = getModelAndViewService().getModelAndViewWithInit(form, ProposalBudgetConstants.KradConstants.BUDGET_DEFAULT_VIEW);
```

Comment:
I can't. This is required regardless because we need to do the 'withInit' part before doing anything else.